app/models/katello/marketing_engineering_product.rb

```diff
@@ -1,9 +0,0 @@
-module Katello
-  class MarketingEngineeringProduct < Katello::Model
```

Comment:
There really is no use for it (same with marketing product) as far as @jlsherrill  and I can tell.